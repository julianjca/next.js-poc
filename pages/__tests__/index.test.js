import shallowRenderer from 'react-test-renderer/shallow'
import renderer from 'react-test-renderer'
import Index from '../index'

const shows = [
  {show: { id: 1, name: 'test'}},
  {show: {id: 2, name: 'test2'}}
]

it('Index should show correct', () => {
  const tree = renderer.create(
    <Index shows={shows} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

it('Index should show correct', () => {
  const renderer = new shallowRenderer
  const snapshot = renderer.render(
    <Index shows={shows} />
  )
  expect(snapshot).toMatchSnapshot()
})
