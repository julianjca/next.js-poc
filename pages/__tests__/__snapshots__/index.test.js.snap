// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should show correct 1`] = `
<div
  style={
    Object {
      "border": "1px solid #DDD",
      "margin": 20,
      "padding": 20,
    }
  }
>
  <div>
    <h2>
      Movie
    </h2>
    <hr />
    <a
      href="/"
      onClick={[Function]}
      style={
        Object {
          "marginRight": 15,
        }
      }
    >
      Home
    </a>
    <a
      href="/about"
      onClick={[Function]}
      style={
        Object {
          "marginRight": 15,
        }
      }
    >
      About
    </a>
  </div>
  <h1>
    My Blog
  </h1>
  <ul>
    <li
      data-jsx={1528442980}
    >
      <a
        data-jsx={1528442980}
        href="/post/1"
        onClick={[Function]}
      >
        test
      </a>
    </li>
    <li
      data-jsx={1528442980}
    >
      <a
        data-jsx={1528442980}
        href="/post/2"
        onClick={[Function]}
      >
        test2
      </a>
    </li>
  </ul>
</div>
`;

exports[`Index should show correct 2`] = `
<Layout>
  <h1>
    My Blog
  </h1>
  <ul>
    <PostLink
      post={
        Object {
          "id": 1,
          "name": "test",
        }
      }
    />
    <PostLink
      post={
        Object {
          "id": 2,
          "name": "test2",
        }
      }
    />
  </ul>
</Layout>
`;
